codebuild/release/validate-prod.yml

```diff
@@ -0,0 +1,18 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      java: $JAVA_ENV_VERSION
+  pre_build:
+    commands:
+      - git clone https://github.com/aws-samples/busy-engineers-document-bucket.git
+      - cd busy-engineers-document-bucket/exercises/java/encryption-context-complete
+  build:
+    commands:
+      - |
+        mvn verify \
+          -Dcheckstyle.skip \
+          -Desdk.version=$VERSION \
+          -Dmaven.compiler.target=$JAVA_NUMERIC_VERSION \
+          -Dmaven.compiler.source=$JAVA_NUMERIC_VERSION
```

Comment:
Are these two properties necessary? Given you're running on either 8 or 11 based on the runtime version config, wouldn't it better represent the common customer experience to stick to the default?

Reply:
They actually are necessary, because the pom currently defaults these to 11. So if you're running in the Java 8 environment but this project is trying to compile to Java 11, it gets unhappy.

Reply:
Ah okay, it's another consequence of using a single consumer to test both version then.